TypeScript.py

```diff
@@ -281,6 +281,9 @@ def __init__(self):
         self.fileMap = {}
         self.refInfo = None
         self.versionST2 = False
+        self.tempFileMap = {}
+        self.tempFileList = []
```

Comment:
tempFileMap is a dictionary mapping request sequence number to the temporary file name used in the request.  tempFileList is a cache of available temporary file names